### PR TITLE
removing : and space between asterik

### PR DESCRIPTION
**Pull request by Neha-Singla**
Screenshot 2024-06-06 at 2 00 28 PM
Screenshot 2024-06-06 at 2 00 35 PM
Screenshot 2024-06-06 at 2 00 45 PM

### CODE REVIEW

**Comment by alisonhall** [comment]
I noticed that capitalization of the inline error message isn't consistent (ie. "enter your phone number", "enter a valid email address", "Phone number invalid"). Maybe we should capitalize the beginning of each of these statements?

File: js/demos/custom-form-example.js [hunk 1]
```diff
@@ -36,8 +36,8 @@ const formValidator = new (function () {
             },
             // Specify validation error messages
             messages: {
-                name_js: 'Error: Please enter your first name',
-                email_js: 'Error: Please enter a valid email address',
+                name_js: 'Error: enter your first name',
+                email_js: 'Error: enter a valid email address',
```
Comment (alisonhall) [inline js/demos/custom-form-example.js:40]: Should we capitalize the 'Enter' word to make it consistent?

**Review by zoltan-dulac — CHANGES_REQUESTED** [review]
Please make alison's change and then I'll approve.

File: js/demos/custom-form-example.js (hunk 1 above)
Comment (Neha-Singla) [reply js/demos/custom-form-example.js]: done

**Review by alisonhall — APPROVED** [review]

**Review by alisonhall — APPROVED** [review]

**Review by zoltan-dulac — APPROVED** [review]

**Review by alisonhall — APPROVED** [review]

File: css/form-error.css [hunk 2]
```diff
@@ -195,7 +195,7 @@
 }
 .enable-form-example label.required::after {
   color: #e60000;
-  content: " *";
+  content: "*";
```
Comment (alisonhall) [inline css/form-error.css:198]: @Neha-Singla , I just noticed that this change was made in the `.css` file, and not the `.less` file. This means that whenever the styles are recompiled, this change is overwritten. All style changes need to be done in the `.less` files.

Comment (Neha-Singla) [reply css/form-error.css:198]: https://github.com/PublicisSapient/enable-a11y/pull/137
Please approve this.

**Pull request merged**